commit-based-cipm/fi-tests/cipm.consistency.fitests.repositorytests/src/cipm/consistency/fitests/repositorytests/util/UtilityClassesRealCodeTest.java

```diff
@@ -0,0 +1,86 @@
+package cipm.consistency.fitests.repositorytests.util;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import cipm.consistency.fitests.repositorytests.util.commentremoval.QuickCommentRemover;
+import cipm.consistency.fitests.repositorytests.util.difffilter.DiffFilter;
+
+/**
+ * A test class that tests {@link QuickCommentRemover} and {@link DiffFilter} on
+ * real GIT diff patches.
+ * 
+ * @author Alp Torac Genc
+ */
+public class UtilityClassesRealCodeTest {
```

Comment:
Both tests in this class fail for me (on Linux) because the Git diffs contain `\r`, which is not removed by the `splitLines` method.